Microsoft.Azure.CosmosRepository/test/Providers/DefaultCosmosItemConfigurationProviderTests.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.CosmosRepository;
+using Microsoft.Azure.CosmosRepository.Options;
+using Microsoft.Azure.CosmosRepository.Providers;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.CosmosRepositoryTests.Providers
+{
+    public class DefaultCosmosItemConfigurationProviderTests
+    {
+        private readonly Mock<ICosmosContainerNameProvider> _containerNameProvider = new();
+        private readonly Mock<ICosmosPartitionKeyPathProvider> _partitionKeyPathProvider = new();
+        private readonly Mock<ICosmosUniqueKeyPolicyProvider> _uniqueKeyPolicyProvider = new();
+
+
+        [Fact]
+        public void GetOptionsAlwaysGetOptionsForItem()
+        {
+            ICosmosItemConfigurationProvider provider = new DefaultCosmosItemConfigurationProvider(
+                _containerNameProvider.Object,
+                _partitionKeyPathProvider.Object,
+                _uniqueKeyPolicyProvider.Object
+            );
```

Comment:
```suggestion
                _uniqueKeyPolicyProvider.Object);
```

Inconsistent styling.

Reply:
Amended.